docroot/themes/custom/govcms_uikit_starterkit/govcms_uikit_starterkit.theme

```diff
@@ -145,3 +145,10 @@ function govcms_uikit_starterkit_preprocess_menu(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_form_element().
+ */
+function govcms_uikit_starterkit_preprocess_form_element(&$variables) {
```

Comment:
@steveworley You have the same code in `docroot/themes/custom/govcms_uikit_base/STARTERKIT/theme-settings.php`. Is it intentional? It seems like it should have been in `STARTERKIT/govcms_uikit_starterkit.theme` instead.